docs_sphinx/user/input.rst

```diff
@@ -155,10 +155,21 @@ Poisson inputs
 If the given value for ``rates`` is a constant, then using
 ``PoissonGroup(N, rates)`` is equivalent to
 ``NeuronGroup(N, 'rates : Hz', threshold='rand()<rates*dt')`` and setting the
```

Comment:
I know that this is not actually your change, but I think it would render better if the `NeuronGroup(...)` line would be on its own (i.e. with `::` and then indented), instead of just quoted.